netlify-plugin-cloudinary/tests/lib/cloudinary.test.js

```diff
@@ -188,6 +191,7 @@ describe('lib/util', () => {
         deliveryType: 'fetch',
         localDir: 'tests',
         remoteHost: 'https://cloudinary.netlify.app',
+        loadingStrategy: 'lazy'
```

Comment:
At line 187: `it('should add lazy loading to image when no option is provided', async () => {`

Since it is no longer possible to call `updateHtmlImagesToCloudinary()` without the `loadingStrategy` parameter, I suggest that we move this into `on-post-build.test.js` where we could verify that the default value is being used if `loadingStrategy` is not specified in `netlify.toml`

Reply:
i think that makes sense! or have it in both spots 🤷

Reply:
Done 🎉 
I removed the test referred to in the above comment and replaced it with 3 new tests, I'll comment on those changes to explain the thought process~